package.json

```diff
@@ -131,7 +127,6 @@
     "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator-for-testing": "^1.0.0",
-    "ember-page-title": "^6.2.2",
```

Comment:
We are not using it yet in the Dummy app, but maybe we should refactor the dummy app to add correct title for each page instead of removing this addon?

Reply:
I agree with removing it, we don't need it.